Sources/Commands/Options.swift

```diff
@@ -315,19 +315,20 @@ public struct SwiftToolOptions: ParsableArguments {
         archs.count > 1 ? .xcode : _buildSystem
     }
     
-    /// Tells `Workspace` to attempt to locate .netrc file at `NSHomeDirectory`.
-    @Flag()
-    var netrc: Bool = false
+    /// Whether to load .netrc files for authenticating with remote servers
+    /// when downloading binary artifacts or communicating with a registry.
+    @Flag(inversion: .prefixedNo, help: "Load credentials from a .netrc file")
+    var netrc: Bool = true
     
-    /// Similar to `--netrc`, but this option makes the .netrc usage optional and not mandatory as with the `--netrc` option.
-    @Flag(name: .customLong("netrc-optional"))
+    @available(*, deprecated, message: ".netrc files are located by default")
```

Comment:
is arg parser smart enough to emit a warning when a flag is marked as deprecated with availability? @natecook1000 ?

Reply:
No, availability information isn't available from within Swift code, so it's something you'd need to emit manually — maybe from a `validate()` method?